modules/dop/endpoints/pipeline.go

```diff
@@ -353,7 +355,23 @@ func (e *Endpoints) pipelineRun(ctx context.Context, r *http.Request, vars map[s
 		IdentityInfo:      identityInfo,
 		PipelineRunParams: runRequest.PipelineRunParams,
 	}); err != nil {
-		return errorresp.ErrResp(err)
+		var apiError, ok = err.(*errorresp.APIError)
+		if !ok {
+			return errorresp.ErrResp(err)
+		}
+
+		ctxMap, ok := apiError.Ctx().(map[string]interface{})
+		if !ok {
+			return errorresp.ErrResp(err)
+		}
+
+		//获取正在运行的流水线链接
+		link, err := GetPipelineLink(p.PipelineDTO, ctxMap)
+		if err != nil {
+			return errorresp.ErrResp(err)
+		}
+
+		return errorresp.ErrResp(apierrors.ErrParallelRunPipeline.InvalidState(link))
```

Comment:
InvalidState(link) enough? I think message should be:

"failed to run pipeline, there is already running: <link>"